src/pages/account/mijn-mappen/[collectionSlug]/index.tsx

```diff
@@ -0,0 +1,297 @@
+			});
+		});
+	};
+
+	/**
+	 * Render
+	 */
+
+	const renderTitleButtons = useMemo(
+		() => [
+			{
+				before: true,
+				node: (
+					<Button
+						className="p-account-my-collections__export--label"
+						variants={['black']}
+						name={t('Metadata exporteren')}
+						label={t('Metadata exporteren')}
+						iconStart={<Icon name="export" />}
+						onClick={(e) => {
+							e.stopPropagation();
+						}}
+					/>
+				),
+			},
+			{
+				before: true,
+				node: (
+					<Button
+						className="p-account-my-collections__export--icon"
+						variants={['black']}
+						name={t('Metadata exporteren')}
+						icon={<Icon name="export" />}
+						onClick={(e) => {
+							e.stopPropagation();
+						}}
+					/>
+				),
+			},
+			...(activeCollection && !activeCollection.isDefault
+				? [
+						{
+							before: false,
+							node: (
+								<Button
+									className="p-account-my-collections__delete"
+									variants={['silver']}
+									icon={<Icon name="trash" />}
+									name={t('Map verwijderen')}
+									onClick={(e) => {
+										e.stopPropagation();
+										setShowConfirmDelete(true);
+									}}
+								/>
+							),
+						},
+				  ]
+				: []),
+		],
+		[t, activeCollection]
+	);
+
+	return (
+		<>
+			<Head>
+				<title>
+					{createPageTitle(
+						t('pages/account/mijn-mappen/index___mijn-mappen') +
+							` | ${activeCollection?.name || collectionSlug}`
+					)}
+				</title>
+
+				<meta
+					name="description"
+					content={t('pages/account/mijn-mappen/index___mijn-mappen-meta-omschrijving')}
+				/>
+			</Head>
+
+			<AccountLayout className="p-account-my-collections">
+				<SidebarLayout
+					color="platinum"
+					sidebarTitle={t(
+						'pages/account/mijn-mappen/collection-slug/index___mijn-mappen'
+					)}
+					sidebarLinks={[
+						...sidebarLinks,
+						{
+							id: 'p-account-my-collections__new-collection',
+							variants: ['c-list-navigation__item--no-interaction'],
+							node: <CreateCollectionButton afterSubmit={collections.refetch} />,
+							hasDivider: true,
+						},
+					]}
+				>
+					{activeCollection && (
+						<>
+							<div className="l-container u-mt-64 u-mb-48">
+								<SidebarLayoutTitle>
+									<EditCollectionTitle
+										key={activeCollection.id}
+										collection={activeCollection}
+										afterSubmit={onCollectionTitleChanged}
+										buttons={renderTitleButtons}
+									/>
+								</SidebarLayoutTitle>
+							</div>
+
+							<div className="l-container u-mb-24:md u-mb-32">
+								<SearchBar
+									backspaceRemovesValue={false}
+									className="p-account-my-collections__search"
+									instanceId="collections-search-bar"
+									light={true}
+									placeholder={t('Zoek')}
+									searchValue={filters.search}
+									onClear={() => {
+										setFilters({
+											search: undefined,
+											page: 1,
+										});
+									}}
+									onSearch={(searchValue: string) => {
```

Comment:
Scenario: 
- zoek op een waarde
- enter
- klik terug op zoekbalk
- enter
De zoekterm verdwijnt uit de zoekbalk, maar blijft wel in de queryparams staan. Kan opgelost worden door current value met nieuwe value te vergelijken en indien ja een rerender te forcen.